api/views/__init__.py

```diff
@@ -146,6 +151,18 @@ class ProfileViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return Profile.objects.filter(user=self.request.user)
 
+    @decorators.action(detail=True, methods=["post"])
+    def report_webcompat_issue(self, request, pk=None):
```

Comment:
Moved the endpoint to its own since the profile value is not being used: [847d735](https://github.com/mozilla/fx-private-relay/pull/2378/commits/847d7352df89960942705bdbe344eb2dd00566f2)